research-software-directory/RSD-as-a-service
# Organisation page design v2

# Organisation page, new design (v2)

Closes #792
Closes #913
Closes #924

Changes proposed in this pull request:
* Organisation page updated according to new design ideas
* Software and Project overview components are made more generic and reused on organisation software and project page
* Set min-height to 60rem for tab content to avoid "jumping" of the content when switching between the tabs
* Upgraded dependencies where possible. There are some issues with latest material-ui version and next.js
* Recovered e2e test with "fake" api calls which seem to be lost during the latest rebasing :-(
* Release date, show message "Release date unknown" when date is not provided


How to test:
* `make start` to rebuild app and generate random data. Notice improved images for software and organisation pages :-)
* Navigate to organisation overview and select one organisation. Examine the content and provide feedback about working
* Login as rsd_admin in order to be able to change any organisation. You should get access to organisation settings, additional ordering options and ability to block or pin software/project items.
* In the settings section you can change basic organisation settings, add/remove maintainers and define about page

## Example organisation page - projects

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/f7647c95-62d5-4149-9871-c19586f8619e)

## Example organisation software - admin options
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/c84bc65f-fa3e-4f66-8b6e-456e13413a57)

## Example organisation settings - admin options
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/f6589a00-2a74-430c-91b4-07de5123449a)

## Example organisation overview card - verified badge
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/197496cb-c2b4-416c-a7a4-cb7a059c5c2c)


PR Checklist:
